security/security-core/src/main/java/io/camunda/zeebe/protocol/record/value/PermissionType.java

```diff
@@ -9,10 +9,17 @@
 
 public enum PermissionType {
   CREATE,
+  CREATE_PROCESS_INSTANCE,
+  CREATE_DECISION_INSTANCE,
+
   READ,
-  READ_INSTANCE,
+  READ_PROCESS_INSTANCE,
   READ_USER_TASK,
+
```

Comment:
@remcowesterhoud it is still missing 
```
READ_DECISION_DEFINITION
READ_DECISION_INSTANCE
UPDATE_DECISION_INSTANCE
CREATE_DECISION_INSTANCE
DELETE_DECISION_INSTANCE
```

Reply:
I only fixed the ones that were being used for verification in Zeebe. I assume anyone who needs others will add them when they require them.